extra-x11/nvidia-prime/autobuild/overrides/usr/share/X11/xorg.conf.d/20-nvidia-prime.conf

```diff
@@ -0,0 +1,4 @@
+Section "ServerLayout"
```

Comment:
Please add some comments describing this configuration's functionalities, and note that it comes from a package (`nvidia-prime`) in this case.